pages/common/openconnect.md

```diff
@@ -0,0 +1,16 @@
+# openconnect
+
+> A VPN client, for Cisco AnyConnect VPNs and others.
+> More information: <https://www.infradead.org/openconnect/>.
+
+- Establish a VPN connection to the specified server:
+
+`openconnect {{vpn.example.org}}`
+
+- Establish a VPN connection to the specified server and continue in the background:
+
+`openconnect --background {{vpn.example.org}}`
+
+- Terminate a VPN session that is running in the background:
```

Comment:
I'm not sure about this example, because a lot of command could have this example. I vote for remote it, but you can wait a second reviewer.
```suggestion
- Terminate the connection that is running in the background:
```

Reply:
There is no other documentation in openconnect on how to terminate a backgrounded session. I personally found this very frustrating, because I had to go online and search for a solution. I would have saved a few minutes if this example was in the man pages, but it was not. 

Apart from that, it is apparently important to do a `killall -SIGINT` instead of a mere `killall`, because without the `-SIGINT` the previous network configuration will not be restored. Even experienced users might not expect this.

Reply:
Oh, I see. Please apply the suggestion and keep the example for now.